### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by Rutger505**

### CODE REVIEW

**Comment by Rutger505** [comment]
To do: Make header responsive, finish tests for header

**Comment by Rutger505** [comment]
Header takes up 1/3 of page...

File: front-end/src/app/layout.tsx [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
 }: Readonly<{
-  children: React.ReactNode;
+  children: ReactNode;
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body
+        className={clsx(
+          "min-h-screen grid grid-cols-[auto_*]",
```
Comment (Rutger505) [inline front-end/src/app/layout.tsx]:
```suggestion
          "min-h-screen grid grid-rows-[auto_1fr]",
```

**Review by JakevdKolk — CHANGES_REQUESTED** [review]
testen gefaald

**Review by JakevdKolk — APPROVED** [review]
tests doen het

**Pull request merged**